Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -124,7 +124,7 @@
   description: uplink-mini-bomb-desc
   productEntity: SyndieMiniBomb
   cost:
-    Telecrystal: 6
+    Telecrystal: 8
```

```suggestion
    Telecrystal: 6
```

With the timer lowered and the original explosion size, should be back down to 6TC.